lib/charms/alertmanager_k8s/v0/alertmanager.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+""" # AlertmanagerConsumer library
+
+This library is design to be used by a charm consuming or providing the alertmanager-k8s relation.
+"""
+
+import ops
+from ops.framework import StoredState, EventSource, EventBase
+from ops.relation import ConsumerBase, ProviderBase
+from ops.charm import CharmBase
+
+from typing import List
+import logging
+
+LIBID = "abcdef1234"  # Unique ID that refers to the library forever
```

Comment:
You will need to make another pull request when you finally set the correct LIBID. Since you are in the process of making a bundle. It may be a good idea to finalize your alertmanager library and then merge this PR.